hw01/hw01.py

```diff
@@ -67,6 +65,13 @@ def largest_factor(n):
     1
     """
     "*** YOUR CODE HERE ***"
+    i=2
+    result=n//i
+    while n%i==0:
+        i+=i
+        return result
```

Comment:
Bad indentation.

And the logic is incorrect.

Reply:
revise indent